pages/genre/_itemId/index.vue

```diff
@@ -98,6 +98,7 @@ export default Vue.extend({
 
       this.loaded = true;
     } catch (error) {
+      console.error(error);
```

Comment:
Add the ignore line here please

Reply:
It's done as I checked

Reply:
@ThibaultNocchi It wasn't before I commented that :)